client/components/jetpack/card/jetpack-product-card-alt-2/features-product-slide-out.tsx

```diff
@@ -31,40 +34,42 @@ const FeaturesProductSlideOut: FunctionComponent< Props > = ( {
 	productBillingTerm,
 	onProductClick,
 } ) => {
+	const { iconSlug, productSlug, displayName, description } = product;
+
 	const siteId = useSelector( getSelectedSiteId );
 	const currencyCode = useSelector( getCurrentUserCurrencyCode );
-	const translate = useTranslate();
+	const purchases = useSelector( ( state ) => getSitePurchases( state, siteId ) );
+	const sitePlan = useSelector( ( state ) => getSitePlan( state, siteId ) );
 
-	// Calculate the product price.
-	const { originalPrice, discountedPrice } = useItemPrice(
-		siteId,
-		product,
-		product?.monthlyProductSlug || ''
-	);
-	const isDiscounted = isFinite( discountedPrice );
-	const productPrice = isDiscounted ? discountedPrice : originalPrice;
-
-	const billingTimeFrame = durationToText( productBillingTerm );
+	const translate = useTranslate();
+	const { originalPrice } = useItemPrice( siteId, product, product?.monthlyProductSlug || '' );
 
-	const slideOutButtonLabel = translate( 'Get {{name/}} $%(price)s', {
-		args: {
-			price: productPrice,
-		},
-		components: {
-			name: <>{ product?.displayName }</>,
-		},
-	} );
+	const isItemPlanFeature = !! ( sitePlan && planHasFeature( sitePlan.product_slug, productSlug ) );
+	const purchase = isItemPlanFeature
+		? getPurchaseByProductSlug( purchases, sitePlan?.product_slug || '' )
+		: getPurchaseByProductSlug( purchases, productSlug );
```

Comment:
I see here we're checking if the product is owned/included as part of the site plan.  Please correct me if I'm wrong, but I think we also need to check if the product is owned as an individual product.  If it's owned as an Individual product, the CTA should point to Manage the _Product_ purchase, and the card badge (at the bottom) should say "_You own this_", If it owned/included as part of an owned Plan, the CTA should should point to manage the _Plan_ purchase, and the card badge at the bottom should say, "_Included in your plan_".